### PR TITLE
Pull request for Git and Bash Profile

### PR DESCRIPTION
**Pull request by Rita-Okonkwo**

### CODE REVIEW

File: command-line/dot-profile.md [hunk 1]
```diff
@@ -74,10 +74,14 @@ of helpful libraries are written using scripts in this way.
...
-It's often helpful to make commands for yourself that are short cuts. For example what if you want to change to
-your directory but you don't want to type `cd ~/scripts`. What if you could just type `cdscr` instead? Let's make a file
-called `aliases` in our scripts folder. 
-type the following:
+It's often helpful to make commands for yourself that are short cuts. Bash aliases allows you to set a shortcut command for a longer command. For example what if you want to change to your directory but you don't want to type `cd ~/scripts`. What if you could just type `cdscr` instead? Aliases are defined in the .bash_profile or .bashrc file typically in your home directory. 
+A bash alias has the following structure:
+
+`alias [alias_name]="[command_to_alias]"`
+
+A new alias always starts on a new line with the **alias** keyword. You define the shortcut command you want to use with the alias name, followed by an equal sign. In quote you type the full command you want to run. This is illustrated below with the "cd" command being customised to "cdscr".
```
Comment (manaswinidas) [inline command-line/dot-profile.md]: Use `cd` instead of "cd" to distinguish this as a command

File: command-line/dot-profile.md [hunk 2]
```diff
@@ -94,6 +98,17 @@ Let's test our alias.
...
+Nano is a Linux command line text editor. It is relatively easy to learn and use. However, it is entirely operated from the keyboard so you will have to learn some keyboard commands. The good news is that the keyboard commands are pretty quick and easy to learn.
+**Things to to take note of when working with nano tool commands:**
+- Any command prefixed with the caret symbol(^) means to use the "ctrl" key. For example, **^G** means to press **ctrl+G** keys at he same time.
+- Any command prefixed with letter M means to press the Alt key. For example, **M-R** means to press **Alt+r** keys together at the same time.
+
+For more detailed info on how to use the nano tool, visit these sites below:
+- [Introduction to the nano text editor](https://staffwww.fullcoll.edu/sedwards/Nano/IntroToNano.html)
+- [A beginners guide to the nano editor](https://www.lifewire.com/beginners-guide-to-nano-editor-3859002)
+
+
```
Comment (manaswinidas) [inline command-line/dot-profile.md]: Remove the extra line

**Comment by Rita-Okonkwo** [comment]
Thanks alot @alodahl @hoduchieu01 @manaswanidas

**Comment by Kundan28** [comment]
@Rita-Okonkwo Thanks for your contribution and support.

**Comment by Rita-Okonkwo** [comment]
It's my pleasure @Kundan28

**Pull request merged**